.github/workflows/release.yaml

```diff
@@ -0,0 +1,36 @@
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - release/v*
```

Comment:
Looking at the tags in webhook, there are no `release/` prefix: https://github.com/rancher/webhook/tags.